api_app/script_analyzers/observable_analyzers/urlscan.py

```diff
@@ -0,0 +1,76 @@
+import requests
+import time
+from api_app.exceptions import AnalyzerRunException
+from api_app.script_analyzers import classes
+from intel_owl import secrets
+
+max_tries: int = 10
```

Comment:
ah, I missed this before. Just a small change: please declare these 2 variables inside the  `__poll_for_result` function only.